From b7b04fc00d6d9edfa0910df834c6a8316c879d16 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:34 -0400
Subject: [redhat] kabi: Adding symbol bio_get_nr_vecs (fs/bio.c)

Message-id: <a8f6caa52ab9564349b4169432e4dacba81eb9be.1410510773.git.poros@redhat.com>
Patchwork-id: 96135
O-Subject: [PATCH RHEL7.1 034/233] kabi: Adding symbol bio_get_nr_vecs (fs/bio.c)
Bugzilla: 1071098
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Return approx number of vecs.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/bio_get_nr_vecs
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/bio_get_nr_vecs
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/bio_get_nr_vecs

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/bio_get_nr_vecs b/redhat/kabi/kabi-module/kabi_ppc64/bio_get_nr_vecs
new file mode 100644
index 0000000..324daa0
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/bio_get_nr_vecs
@@ -0,0 +1,2 @@
+#1-
+0xe1d1a963 bio_get_nr_vecs vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/bio_get_nr_vecs b/redhat/kabi/kabi-module/kabi_s390x/bio_get_nr_vecs
new file mode 100644
index 0000000..3777e54
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/bio_get_nr_vecs
@@ -0,0 +1,2 @@
+#1-
+0x8e9d1f16 bio_get_nr_vecs vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/bio_get_nr_vecs b/redhat/kabi/kabi-module/kabi_x86_64/bio_get_nr_vecs
new file mode 100644
index 0000000..9edd70d
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/bio_get_nr_vecs
@@ -0,0 +1,2 @@
+#1-
+0x37be9f32 bio_get_nr_vecs vmlinux EXPORT_SYMBOL
-- 
1.7.1